js/ui/appDisplay.js

```diff
@@ -1091,6 +1091,9 @@ const FolderView = new Lang.Class({
         this.actor = new St.ScrollView({ overlay_scrollbars: true });
         this.actor.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC);
         let scrollableContainer = new St.BoxLayout({ vertical: true, reactive: true });
+        this.no_apps_label = new St.Label({ text: _('No apps in this folder! To add an app, drag it onto the folder.'),
```

Comment:
Please use camel case -- there should also be no reason to make this "public".
Finally, the coding style uses double quotes for translatable strings.